bundle/src/main/resources/CQ-INF/nodetypes/cqsm_nodetypes.cnd

```diff
@@ -1,3 +1,22 @@
+/*-
+ * ========================LICENSE_START=================================

```

Comment:
Comments
Comment can also be included in the notation using either of the standard Java forms:
```
// A comment
/* Another comment */
```